test-lib.sh

```diff
@@ -765,20 +801,6 @@ EOF
     )
 }
 
-# ct_check_image_availability PUBLIC_IMAGE_NAME
-# ----------------------------
-# Pull an image from the public repositories to see if the image is already available.
-# Argument: PUBLIC_IMAGE_NAME - string containing the public name of the image to pull
-ct_check_image_availability() {
-  local public_image_name=$1;
-
-  # Try pulling the image to see if it is accessible
-  if ! docker pull "$public_image_name" &>/dev/null; then
-    echo "$public_image_name could not be downloaded via 'docker'"
-    return 1
-  fi
-}
-
```

Comment:
Can we keep the function and just call `ct_pull_image` inside? I would rather not remove the function completely in case it is used in some of the repositories

Reply:
Sure. I will do it.